applications/asset_tracker_v2/src/modules/data_module.c

```diff
@@ -604,6 +615,30 @@ static void data_encode(void)
 	}
 }
 
+static void data_agps_request_encode(void)
+{
+	int err;
+	struct cloud_codec_data codec = {0};
+
+	err = cloud_codec_encode_agps_request(&codec, &agps_request);
+	switch (err) {
+	case 0:
+		LOG_DBG("AGPS request encoded successfully");
+		data_send(DATA_EVT_AGPS_REQUEST_DATA_SEND, AGPS_REQUEST, &codec);
+		break;
+	case -ENOTSUP:
+		/* AGPS request encoding not supported */
```

Comment:
Why not log that as well?

Reply:
Probably an overlook. I've added a log message with warning log level.